.github/workflows/trunk.yml

```diff
@@ -501,7 +501,7 @@ jobs:
       docker-image: executorch-ubuntu-22.04-qnn-sdk
       submodules: 'recursive'
       ref: ${{ github.event_name == 'pull_request' && github.event.pull_request.head.sha || github.sha }}
-      timeout: 900
+      timeout: 90
```

Comment:
Hey sorry miss this PR. I thought I left comment. Is there any particular reason that we lower the timeout? I think the CI machine is not very powerful, and I'm worried that it will cause timeout again, and will block updating the strict branch. Should we remove this change and see what happens?